cypress/components/select/simple-select/simple-select.cy.tsx

```diff
@@ -972,18 +972,35 @@ context("Tests for SimpleSelect component", () => {
       selectList().should("not.be.visible");
       commonDataElementInputPreview().should("not.be.focused");
     });
+
+    it.only("should be able to focus the last item in the select list when the select list has an OptionGroupHeader ", () => {
```

Comment:
```suggestion
    it("should be able to focus the last item in the select list when the select list has an OptionGroupHeader ", () => {
```

This means only this test runs from this file, and no others.

Going to assume you added this for fast iteration when writing the test and forgot to remove it before committing - but either way it needs removing now.

Reply:
Yep. Well caught 😄